addons/spreadsheet/static/src/global_filters/plugins/global_filters_core_view_plugin.js

```diff
@@ -63,6 +55,9 @@ export class GlobalFiltersCoreViewPlugin extends OdooCoreViewPlugin {
     allowDispatch(cmd) {
         switch (cmd.type) {
             case "SET_GLOBAL_FILTER_VALUE": {
+                if (cmd.displayNames) {
+                    throw new Error("Display names");
+                }
```

Comment:
Is it leftover dev code ? If it's intended, don't throw in an allowDispatch, return CommandResults 🙃

Reply:
🙈